cohort_schedules/C07_classroom.md

```diff
@@ -1,39 +1,126 @@
-| Jul 3   | Week 22 | Capstone Project
-| Jul 10  | Week 23 | Capstone Project
-| Jul 17  | Week 24 | Capstone Project
-| Jul 24  | Week 25 | Capstone Project Presentations
-| Jul 31  | Week 26 | _BREAK_
-| Aug 7   | Week 27 | __INTERNSHIP BEGINS__
+# Ada Developers Academy Cohort 7 Syllabus
+
+## Overview
+
+Ada Developers Academy teaches full-stack web development, allowing students to choose what specialty of software development they want to pursue in their post-Ada careers. We cover HTML/CSS, JavaScript, Ruby on Rails, and computer science fundamentals. Our [complete curriculum](https://github.com/Ada-Developers-Academy/textbook-curriculum/) is online and open-source.
+
+We focus on teaching skills that are readily transferrable from one technology stack to another, making Ada students adaptable and flexible candidates for all types of developer positions. Students have been successfully placed in internships and jobs working in Ruby, Java, JavaScript, C++, C#, Objective-C and Python, among other languages and frameworks.
+
+Major topics of study include:
+
+| Writing Software        | Being a Developer
+|-------------------------|-------------------
+| Ruby                    | Leadership & Inclusion
+| HTML & CSS              | Test-Driven Development
+| Ruby on Rails           | Pair Programming
+| Web APIs                | Agile Practices
+| JavaScript              | Networking
+| CS Fundamentals         | Career Readiness
+
+## Expected Outcomes
+
+### Ruby
+
+Ruby is a dynamically typed, interpreted, object-oriented language. In addition to serving as a great introduction to programming, Ruby is the foundation of the Ruby on Rails framework, and is used for most of our CS Fundamentals curriculum.
+
+Students should be able to:
+
+* Write complex command line programs in Ruby
+* Use object-oriented programming principals to build scalable, maintainable programs
+* Process and build relations between data from files
+* Verify code from the outset using test-driven development
+
+### HTML & CSS
+
+HTML and CSS are two of the core technologies on which the web is built, allowing web developers to structure content in a hierarchy and specify how it should be presented to the user. Both are foundational for much of the rest of the course.
+
+Students should be able to:
+
+* Read and write HTML and CSS
+* Utilize semantic tags to provide clear structure to HTML
+* Manage the layout of a complex web page using CSS
+
+### Ruby on Rails
+
+Ruby on Rails is a robust and comprehensive MVC framework for building web servers. We use Rails to address a wide range of topics, from HTTP, REST and web standards to database operations. This unit also includes a large team project completed over two weeks.
+
+Students should be able to:
+
+* Build and deploy a complete website, from the HTML & CSS on the page to the backing database
+* Model complex real-world relations using ActiveRecord
+* Understand the basics of web security and user authentication
```

Comment:
Do they understand the basics of web security? Is there something specific you're referencing here?